openmsupply/application-manager-web-app
# #468 bug fixes from demo part 1

### Changes

* Fixed Re-Review(0) issue
* Fixes issue with review progress not considering visibility

### Testing 

Use FE-helpers-for-bug-fixes-from-demo back 

* Start ABC123 review with reviewer 1, approve all page 2 question, making sure to not use approve all on last page, progress should show fully completed (correctly)
* Submit LOQ, make changes by applicant (make sure one section is not changed), Re-Review(0) should not appear in review home
